components/date-picker/generatePicker/generateSinglePicker.tsx

```diff
@@ -137,6 +145,7 @@ export default function generatePicker<DateType>(generateConfig: GenerateConfig<
             superNextIcon={<span className={`${prefixCls}-super-next-icon`} />}
             allowClear
             transitionName={`${rootPrefixCls}-slide-up`}
+            format={locale!.dateFormat}
```

Comment:
`dateFormat ` needs to be implemented in rc-piker

Reply:
rc-piker：https://github.com/react-component/picker

Reply:
oh, I see, I misunderstand the discussion, let me see if I can fix it in the rc-picker side and then bump it here!

Do you happen to know the commit in which the issue occurred in the rc-picker?

Would it be better to close this pull request or save it for bump/upgrade?

Reply:
It should be a refactoring that left this logic. I'll take time to find the relevant pr.